fido2.go

```diff
@@ -675,6 +687,9 @@ func (d *Device) Assertion(
 
 	// Get assertion
 	if cErr := C.fido_dev_get_assert(dev, cAssert, cStringOrNil(pin)); cErr != C.FIDO_OK {
+		if cErr == C.FIDO_ERR_KEEPALIVE_CANCEL {
```

Comment:
Don't need this since errFromCode does it.

Reply:
If we remove that the error will be wrapped so you need to unwrap on the client side to check if it was interrupted.
What do you think is the best approach here?

Reply:
I think it is ok to expect callers should check errors with errors.Cause(err).